rubocop/cop/i18n/deprecated_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module I18n
+      # Checks for uses of the deprecated helper function, `ts`.
+      # Strings passed to it cannot be translated, and all calls
+      # will need to be replaced with I18n.t to enable UI translations
+      # in the future.
+      #
+      # @example
+      #  # bad
+      #  ts("This will only be in English :(")
+      #  ts("Hello %{name}", name: "world")
+      #
+      # @example
+      #  # good
+      #  t(".relative.path.to.translation")
+      #  t(".greeting", name: "world")
+      class DeprecatedHelper < RuboCop::Cop::Base
+        MSG = "Prefer Rails built-in `I18n.t` function over `ts`: the latter is not actually translatable"
```

Comment:
Can this link to our GitHub wiki page about i18n standards?

Reply:
Done! (I wanted to make this multi-line with a heredoc, but the rubocop tests get upset at that and I think it's OK to be all one line so :shrug: )